gobblin-core/src/main/java/org/apache/gobblin/converter/avro/JsonElementConversionWithAvroSchemaFactory.java

```diff
@@ -183,7 +189,7 @@ public Schema schema() {
     WorkUnitState state;
 
     public RecordConverter(String fieldName, boolean nullable, String sourceType, Schema schemaNode,
-        WorkUnitState state, List<String> ignoreFields) throws UnsupportedDateTypeException {
```

Comment:
No. Just because inside this method, nothing will throw that exception. Seems like useless code so I remove it

Reply:
are you sure super() won't throw such exception at all?

Reply:
Yes, super() will not throw this exception. The other constructers throw out this exception is because getConvertor() method may throw this exception